apps/tlon-mobile/src/screens/ChannelScreen.tsx

```diff
@@ -29,6 +29,16 @@ export default function ChannelScreen(props: ChannelScreenProps) {
       }
     }, [props.route.params.channel.group])
   );
+  useFocusEffect(
+    useCallback(
+      () =>
+        // Mark the channel as visited when we unfocus/leave this screen
+        () => {
+          store.markChannelVisited(props.route.params.channel);
```

Comment:
is this the only place that we'd need to mark a channel as visited? (on that note, can you think of a more specific term than "visited?" we are not considering any view other than the main `ChannelScreen` as a visit. I can't!)

Reply:
Nope, that makes sense to me! `ChannelScreen` really is the only context in which you view a channel. And visited works for me, or `viewed` to match the sqlite column.